source/extensions/omni.isaac.lab/omni/isaac/lab/controllers/operational_space.py

```diff
@@ -185,43 +134,83 @@ def reset_idx(self, robot_ids: torch.Tensor = None):
         """Reset the internals."""
         pass
 
-    def set_command(self, command: torch.Tensor):
+    def set_command(self, command: torch.Tensor, ee_pose: torch.Tensor | None = None):
         """Set target end-effector pose or force command.
 
         Args:
             command: The target end-effector pose or force command.
         """
         # check input size
-        if command.shape != (self.num_robots, self.num_actions):
+        if command.shape != (self.num_envs, self.action_dim):
             raise ValueError(
-                f"Invalid command shape '{command.shape}'. Expected: '{(self.num_robots, self.num_actions)}'."
+                f"Invalid command shape '{command.shape}'. Expected: '{(self.num_envs, self.action_dim)}'."
             )
         # impedance mode
         if self.cfg.impedance_mode == "fixed":
             # joint positions
             self._task_space_target[:] = command
```

Comment:
if this joint positions, then I think it would be better to name this `_joint_space_target`

Reply:
It shouldnt be joint positions but rather task space targets. I have changed changed all accordingly.